src/status_im/chat/events.cljs

```diff
@@ -161,10 +161,7 @@
   [{:keys [chats] :accounts/keys [current-account-id] :as db} existing-account?]
   (if (chats const/console-chat-id)
     {:db db}
-    (cond-> {:db (-> db
-                     (assoc :new-chat sign-up/console-chat
-                            :current-chat-id const/console-chat-id)
-                     (update :chats assoc const/console-chat-id sign-up/console-chat))
+    (cond-> {:db (update db :chats assoc const/console-chat-id sign-up/console-chat)
```

Comment:
Can you align map values?

Reply:
Why is new-chat and current-chat-id no longer used? Can you please elaborate?